packages/venia-concept/webpack.config.js

```diff
@@ -1,261 +1,42 @@
-                        if (name.startsWith('RootCmp')) {
-                            const filenames = Array.isArray(value)
-                                ? value
-                                : [value];
-                            assets.bundles.prefetch.push(...filenames);
-                        }
-                        const ext = path.extname(name);
-                        const type = ext && ext.replace(/^\./, '');
-                        if (type) {
-                            if (!assets[type]) {
-                                assets[type] = {};
-                            }
-                            assets[type][path.basename(name, ext)] = value;
-                        }
-                    });
-                }
-            })
+const { configureWebpack } = require('@magento/pwa-buildpack');
+
+module.exports = async env => {
+    const config = await configureWebpack({
+        context: __dirname,
+        vendor: [
+            'apollo-cache-inmemory',
+            'apollo-cache-persist',
+            'apollo-client',
+            'apollo-link-context',
+            'apollo-link-http',
+            'informed',
+            'react',
+            'react-apollo',
+            'react-dom',
+            'react-feather',
+            'react-redux',
+            'react-router-dom',
+            'redux',
+            'redux-actions',
+            'redux-thunk'
         ],
-        optimization: {
-            splitChunks: {
-                cacheGroups: {
-                    vendor: {
-                        test: new RegExp(
-                            `[\\\/]node_modules[\\\/](${libs.join('|')})[\\\/]`
-                        ),
-                        chunks: 'all'
-                    }
-                }
+        special: {
+            '@magento/peregrine': {
+                esModules: true,
+                cssModules: true
             }
-        }
-    };
-    if (mode === 'development') {
-        config.devtool = 'eval-source-map';
-        config.devServer = await PWADevServer.configure({
-            publicPath: config.output.publicPath,
-            graphqlPlayground: true,
-            ...projectConfig.sections(
-                'devServer',
-                'imageService',
-                'customOrigin'
-            ),
-            ...projectConfig.section('magento')
-        });
-
-        // A DevServer generates its own unique output path at startup. It needs
-        // to assign the main outputPath to this value as well.
+            // '@magento/venia-library': {
+            //     cssModules: true,
+            //     esModules: true,
+            //     graphQLQueries: true,
+            //     rootComponents: true,
+            //     upward: true
+            // }
+        },
+        env
+    });
 
-        config.output.publicPath = config.devServer.publicPath;
+    config.module.noParse = [/braintree\-web\-drop\-in/];
```

Comment:
Will this have an affect on runtime performance? Hard for me to tell if we need this or not as `stage:venia` is broken for me :(